### PR TITLE
fix(containers) containers being destroyed should skip afterRender task

### LINKED ISSUE #101: Uncaught TypeError: Cannot read property 'clientWidth' of null at monitor.js
A component in state "destroying" is added to monitoring at https://github.com/html-next/flexi/blob/master/addon/mixins/container.js#L45 

It has element = null, which leads to the issue later on. Because null.clientwidth is expected to exist here: https://github.com/html-next/flexi/blob/master/addon/lib/monitor.js#L75

screenshot

### PR DESCRIPTION
**Pull request by kybishop**
Fixes #101

### CODE REVIEW

File: tests/integration/components/flexi-container-test.js [hunk 1]
```diff
@@ -87,6 +87,14 @@ test('huge responsive containers are responsive', function(assert) {
   assert.ok(getElement(this).className.includes('container-xs'), 'We rendered the right classes for mobile');
 });
 
+test('it does not attempt to monitor containers in the isDestroying state', function(assert) {
+  this.render(hbs`
+    {{component "tests/components/destroyed-container"}}
```
Comment (kybishop) [inline tests/integration/components/flexi-container-test.js]: Without the fix, this throws an error and the test fails.

File: tests/dummy/app/routes/tests/components/destroyed-container/template.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{{#if showContainer}}
+  {{#flexi-container}}
+    template block text
+  {{/flexi-container}}
+{{/if}}
+
+{{#component "tests/components/destroyed-container/components/component-that-destroys-container" showContainer=(mut showContainer)}}
```
Comment (rwwagner90) [inline tests/dummy/app/routes/tests/components/destroyed-container/template.hbs]: Can't this be `{{#destroyed-container/component-that-destroys-container`?

Comment (kybishop) [inline tests/dummy/app/routes/tests/components/destroyed-container/template.hbs]: removed the component helper from here and the test itself, though the names are still a little longer than hoped for.

**Comment by rwwagner90** [comment]
Looks good to me after the changes!

**Pull request merged**